package.json

```diff
@@ -36,13 +39,14 @@
   "scripts": {
     "debug": "sails debug",
     "start": "node app.js",
-    "test": "NODE_ENV=test node ./node_modules/mocha/bin/mocha test/bootstrap.test.js test/**/*.test.js"
+    "test": "NODE_ENV=test node ./node_modules/mocha/bin/mocha test/bootstrap.test.js test/**/*.test.js",
+    "lint": "eslint test/**/*.js api/**/*.js"
   },
   "main": "app.js",
   "repository": {
     "type": "git",
     "url": "git://github.com/sergii/quicksilver-api.git"
```

Comment:
Thanks for noticing it! `package.json` was generated by `sails new`, and somehow it used my fork instead of DoSomething org.